deepchem/utils/openvino_model.py

```diff
@@ -8,17 +8,62 @@
 
 import numpy as np
 
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
 import tensorflow as tf
 from tensorflow.python.framework.convert_to_constants import convert_variables_to_constants_v2
 
 try:
   import mo_tf
-  from openvino.inference_engine import IECore, StatusCode, ExecutableNetwork
+  from openvino.inference_engine import IECore, StatusCode, ExecutableNetwork, IENetwork
   is_available = True
 except:
   is_available = False
 
 
+def softplus(x):
+  return torch.where(x < np.log(np.finfo(np.float32).max),
+                     torch.log(torch.exp(x) + 1), x)
+
+
+def pooling(graph, feat):
+  seglen = tuple(graph.batch_num_nodes(None).detach().numpy())
+  value = torch.split(graph.ndata[feat], seglen)
+  return torch.stack([torch.mean(inp, 0) for inp in value])
+
+
+class CGCNN_OV(nn.Module):
+  """Version of Crystal Graph Convolutional Neural Network (CGCNN) adapted for conversion to IR."""
+
+  def forward(self, inputs):
+    from deepchem.models.torch_models.cgcnn import CGCNNLayer
+    graph = self.model.graph
+    def message_func(self, edges):
+      srcdata = graph._node_frames[0].subframe(inputs[2])['x']
+      dstdata = graph._node_frames[0].subframe(inputs[3])['x']
```

Comment:
Can you please show ONNX model visualization (top layers)

Reply:
![image](https://user-images.githubusercontent.com/32199725/114070107-5866ec80-98a8-11eb-93ff-373856cbe1f9.png)